lib/streams/json-parse-stream.js

```diff
@@ -0,0 +1,24 @@
+var Parser = require('jsonparse');
```

Comment:
There's no need to include an external parser for what this stream does.  All it's doing right now is parsing whole strings containing objects.  You can just use JSON.parse() for this.

See my note below about this not being an actual JSON parsing stream.  A streaming parser would be useful for an actual JSON parsing stream.